pkg/destroy/aws/aws.go

```diff
@@ -2021,3 +2079,54 @@ func removeSharedTag(ctx context.Context, tagClients []*resourcegroupstaggingapi
 
 	return nil
 }
+
+// destroyedResources stores the results of attempts to destroy resources. The key is the ARN of the resource. The value
+// is whether the resource was successfully destroyed.
+type destroyedResources map[string]bool
```

Comment:
Is is useful to build our own implementation of set here? Could we just use https://godoc.org/k8s.io/apimachinery/pkg/util/sets ?

Reply:
The `destroyedResources` type is no longer being used and has been removed.